Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/StartRide/Internal.hs

```diff
@@ -31,10 +31,10 @@ import qualified Storage.Queries.DriverInformation as SQD
 import qualified Storage.Queries.Ride as QRide
 import Tools.Event
 
-startRideTransaction :: (MonadFlow m, CacheFlow m r, EsqDBFlow m r, EventStreamFlow m r, LT.HasLocationService m r) => Id SP.Person -> SRide.Ride -> Id SRB.Booking -> LatLong -> Id Dmerch.Merchant -> Maybe SRide.OdometerReading -> m ()
+startRideTransaction :: (MonadFlow m, CacheFlow m r, EsqDBFlow m r, EventStreamFlow m r, LT.HasLocationService m r, HasShortDurationRetryCfg r c) => Id SP.Person -> SRide.Ride -> Id SRB.Booking -> LatLong -> Id Dmerch.Merchant -> Maybe SRide.OdometerReading -> m ()
 startRideTransaction driverId ride bookingId firstPoint merchantId odometer = do
   triggerRideStartEvent RideEventData {ride = ride{status = SRide.INPROGRESS}, personId = driverId, merchantId = merchantId}
-  void $ LF.rideStart ride.id firstPoint.lat firstPoint.lon merchantId driverId
```

Comment:
@ratnadeep-99 Can you add this inside all LTS APIs Flow file `rideStart, rideEnd, rideDetails, nearBy`